submissions/phpslonuk/JS_DOM/script/main.js

```diff
@@ -0,0 +1,116 @@
+	},
+	{
+		image: "https://raw.githubusercontent.com/phpslonuk/JS_DOM/main/img/England.jpg",
+		title: "Різдво в Англії",
+		alt: "England",
+		description: "В Англії Різдво – справжнє сімейне свято. Цього дня всі збираються в батьківському домі, дарують один одному подарунки і переглядають сімейні фотографії. Традиційною стравою є індичка під соусом з аґрусу і пудинг. А п’ють в цей день виключно чай або бренді. Окрасою англійського різдвяного столу є святковий пиріг. За старовинним звичаєм, його начиняють різними предметами, що пророкують долю на найближчий рік тому, в чиєму шматку виявиться цей предмет. Англійці вірять, що каблучка означає швидке весілля, копійка – багатство в майбутньому, а підкова – успіх в новому році."
+	},
+	{
+		image: "https://raw.githubusercontent.com/phpslonuk/JS_DOM/main/img/Japan.jpg",
+		title: "Різдво в Японії",
+		alt: "Japan",
+		description: "В Японії замість Діда Мороза основна фігура свята – бог Хотейошо, у якого очі… на потилиці. Традиційне різдвяне меню японців – рис, боби, гриби і бамбук. І, звичайно, улюблений веселий напій саке. Ще на Різдво японці біля своїх будинків розвішують на високих жердинах пучки соломи і підпалюють їх, щоб відлякати злих духів."
+	},
+	{
+		image: "https://raw.githubusercontent.com/phpslonuk/JS_DOM/main/img/USA.jpg",
+		title: "Різдво в США",
+		alt: "USA",
+		description: "Сполучені штати – багатонаціональна країна, тому можна побачити різні способи святкування Різдва. Деякі готують традиційну фаршировану індичку. Інші вважають, що куди  краще ковбаски, страви з капусти та супи. Італійці готують на Різдво лазанью. Діти отримують подарунки вранці. Їм кажуть, що вночі приходив Санта-Клаус, він спустився димоходом і приніс подарунки. У різдвяний ранок традиційно подають рулет з корицею і каву, а на обід – шинку. Американці – нація, яка чи не найбільше отримує і дарує подарунків у це свято. У невеликих містах Америки прийнято ходити один до одного в гості та всім разом співати різдвяні пісні. У США будь-які заклади,  чи то банк, магазин чи фірма, зачиняються. Всі відпочивають і найчастіше дивляться по телевізору «Лускунчика» і «Червону шапочку»."
+	}
+]
+
+ELEMENTS.forEach(elem => {
+	elem.addEventListener("click", getElementAttribute)
+})
+
+function doWithElement(db, index) {
+	TITLE.innerHTML = db[index].title;
+	COUNTRY_IMG.src = db[index].image;
+	COUNTRY_DESCRIPTION.innerHTML = db[index].description;
+}
+
+function getElementAttribute(e) {
+	let index;
+	if (e.target.nodeName == 'A') {
+		index = e.path[1].attributes[1].nodeValue;
+	} else {
+		index = e.target.attributes[1].nodeValue;
+	}
+
```

Comment:
That is unnecessary, you can get your data attribute from event target: `e.target.dataset[YOUR_DATA_ATTRIBUTE_NAME]`